src/visualization/include/iDynTree/MeshcatVisualizer.h

```diff
@@ -58,16 +58,21 @@ class MeshcatVisualizer
                        const iDynTree::Span<const double> &jointPositions,
                        const std::string &modelName);
 
+    bool set_sphere(const std::string &name,
```

Comment:
For consistency with the rest of the class can you use camelcase naming and add a documentation block?